tests/test_reproducible_wheels.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+import subprocess
+
+
+# These are the SDW repositories that we build wheels for.
+REPOS_WITH_WHEELS = [
+    "securedrop-client",
+    "securedrop-log",
+    "securedrop-proxy",
+]
```

Comment:
Are there other repositories for which we build wheels, that should be added to this list?